pkg/meta/tkv.go

```diff
@@ -1491,10 +1488,6 @@ func (m *kvMeta) doRename(ctx Context, parentSrc Ino, nameSrc string, parentDst
 	var dtyp uint8
 	var tattr Attr
 	var newSpace, newInode int64
-	lockParent := parentSrc
-	if isTrash(lockParent) {
```

Comment:
we should not lock the source if it's trash, otherwise the `restore --putback --threads 10` will be slow

Reply:
OK, updated. I didn't recognize this optimization...